modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/_question.scss

```diff
@@ -0,0 +1,17 @@
+.question-headline {
+	margin-bottom: 0;
+}
+
+.question-answer {
+	border-top: 1px solid #E5E5E5;
+	padding: 0.5em;
```

Comment:
Rem units :) https://j.eremy.net/confused-about-rem-and-em/

Reply:
Got it! https://github.com/boton/liferay-portal/pull/123/commits/991a498c88fdbf9e652e29c65ca4cac84816de37 Thank you so much :)